include/GIL/InstBase.hpp

```diff
@@ -428,6 +428,118 @@ class TerminatorInst : public InstBase {
     }
 };
```

Comment:
question is, do we want to put all instructions in the same file? (Swift does put everything in one file)